scripts/v6/prime/06-asset-admin.sql

```diff
@@ -4,6 +4,4 @@ INSERT INTO AssetRefCounter (counter_name, counter_prev_id, counter_current_id,
 INSERT INTO AssetRefCounter (counter_name, counter_prev_id, counter_current_id, counter_next_id) VALUES ('scoaPostingId', NULL,0,1);
 
 -- AssetFinYear inserts
-INSERT INTO AssetFinYear (FinYear, Period) VALUES(2015,1);
-INSERT INTO AssetFinYear (FinYear, Period) VALUES(2016,1);
-INSERT INTO AssetFinYear (FinYear, Period) VALUES(2017,1);
\ No newline at end of file
+INSERT INTO AssetFinYear (FinYear, Period) VALUES(2015,1);
```

Comment:
I think  these were used for specific test cases. But this is the correct way imo. The tests using the other values must insert them.